ckanext/resource_type_validation/resources/resource_types.json

```diff
@@ -13,18 +13,26 @@
             "application/sparql-query",
             "application/x-ascii-grid",
             "application/xml",
+            "application/json",
             "application/rdf+xml",
+            "application/vnd.google-earth.kml+xml",
             "text/*"
         ],
         "application/zip": ["application/vnd.google-earth.kmz"],
         "application/CDFV2": [
             "application/vnd.ms-excel",
             "application/msword",
             "application/vnd.ms-powerpoint"
+        ],
+        "application/CDFV2-unknown": [
+            "application/vnd.ms-excel",
+            "application/msword",
+            "application/vnd.ms-powerpoint"
         ]
     },
     "equal_types": [
-        ["text/xml", "application/xml", "application/rdf+xml"]
+        ["text/xml", "application/xml", "application/rdf+xml"],
+        ["text/xml", "application/xml", "application/vnd.google-earth.kml+xml"]
```

Comment:
This isn't ideal, since we could lose information; a KML file might end up being shown as merely XML because the system can't tell that one is more specific than the other. Is it possible to use an override instead?

Reply:
That makes sense, but I'm having trouble getting it to pass the tests.
I tried using the override
`"application/xml": ["application/vnd.google-earth.kml+xml"]`
but then I get an assertion error for test_coalesce_candidates:
`AssertionError: 'text.plain' != 'application/xml'`

Reply:
Your changes look good, and are passing locally for me as well as on GitHub.